### ISSUE TITLE
Google Pay and Apple Pay not showing Automatically

### ISSUE THREAD

**Issue by uzairahim** [vue-stripe/vuestripe.com#3]
I've implemented the stripe-checkout but not able to see Google Pay Button on Payment screen (using chrome) and same with Apple pay (using Safari)

here is my component code.
<stripe-checkout
                  ref="checkoutRef"
                  :pk="publishableKey"
                  :amount="amount"
                  @token="tokenCreated"
                  @loading="loading = $event"
                  submitType="pay"
                >
</stripe-checkout>

**Comment by jofftiquez** [comment]
Hello this is not something specific to this plugin, but in the stripe's checkout itself. As you can see, there are no configs needed in this plugin to achieve that.

In your chrome, do you have a google account logged? And does that account have a confirmed payment method?

**Comment by jofftiquez** [comment]
I'm closing this because this is not something that can be fixed within this plugin. But I am willing to help, just post your replies here. Thanks.

**Comment by benedict-w** [comment]
I'm having this issue too, using the `<stripe-element-card>` implementation

Have upgraded vue-stripe to `"@vue-stripe/vue-stripe": "^4.2.5"`

In Stripe dashboard we have _"Apple Pay is live"_ on the enabled payment tabs. I have also uploaded the apple domain verification file.

I've tried in Safari and Chrome but we only get the card input field and no other payment buttons. I have tried other sites where we know these to work (not using vue-stripe) and the buttons display, so I know my browsers are configured for payment.

Is there anything else we can do to check the implementation of vue-stripe?
